roles/sap_vm_provision/defaults/main.yml

```diff
@@ -10,6 +10,8 @@ sap_vm_provision_iac_type: ""
 # aws_ec2_vs , gcp_ce_vm , ibmcloud_vs , ibmcloud_powervs , msazure_vm , ibmpowervm_vm , kubevirt_vm , ovirt_vm , vmware_vm
 sap_vm_provision_iac_platform: ""
 
+# jumphost where ansible playbook will delegate_to
```

Comment:
Update comment to explain that this is not related to cloud platforms. Example:
```
# kubevirt_vm: Jumphost where ansible playbook will delegate_to
# This is not used by cloud platforms, because they use Bastion hosts.
```